dnf-behave-tests/dnf/transaction-sr/replay.feature

```diff
@@ -1140,6 +1140,9 @@ Given I successfully execute dnf with args "install archchange-1.0"
       | top-a-1:1.0-1.x86_64    | User            |
 
 
+# This should fail or there should be at least a warning.
+# Reported as https://github.com/rpm-software-management/dnf5/issues/1573
+@xfail
```

Comment:
I was a bit confused by this, because the `xfail` tests are usually written according to the expected behavior, but here there is the undesired (though currently working) behavior. Based on the discussion in the issue, I think dnf should report an error. Do I understand it correctly? If so, can you please change the expected outcome?

Reply:
Yes, sure, updated.